### PR TITLE
Update logsearch restore doc

### PR DESCRIPTION
**Pull request by markdboyd**
## Changes proposed in this pull request:

- Various additions and improvements to docs about reindexing data to Elasticsearch from S3 that were identified during teh quarterly backup exercise

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/update-logsearch-restore-doc)

## Security Considerations

None. Just improving documentation.

### CODE REVIEW

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 1]
```diff
@@ -62,15 +62,89 @@ ElasticSearch is corrupted or lost.
...
+To determine what value to use for `${aws_partition}`, view the bucket ARN
+in the S3 console.
+
+Save the new role policy and the VM should now have the appropriate permissions
+to fetch backups from S3 for ingestion.
 
 ### Create a custom logstash.conf for the restore operation
 
-Log into any `ingestor` node and make a copy of its current logstash configuration.
+From the jumpbox of the environment where you want to do the reindexing, log into
```
Comment (jameshochadel) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: Thoughts on writing out the jumpbox command here?

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 2]
```diff
@@ -84,21 +158,16 @@ input {
...
 }
 ```
 
-The values for `:bucket:` and `:region:` can be found in [cg-provision](https://github.com/18F/cg-provision/blob/main/terraform/modules/cloudfoundry/buckets.tf#L25-L30)
-or retrieved from the bosh manifest:
-
-```sh
-bosh -d logsearch manifest
-```
+The values for `:bucket:` and `:region:` can be retrieved from the bosh manifest
```
Comment (jameshochadel) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: Did we want to maybe move this earlier in the sequence so the operator doesn't have to either 1) jump into and out of their `bosh ssh` session on the ingestor or 2) create a second jumpbox session to grab the values?

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 3]
```diff
@@ -349,20 +415,26 @@ output {
 }
 ```
 
-The values for `:bucket:` and `:region:` can be found in [cg-provision](https://github.com/18F/cg-provision/blob/master/terraform/modules/cloudfoundry/buckets.tf#L25-L30)
-or retrieved from the bosh manifest via `bosh -d logsearch manifest`.
+The values for `:bucket:` and `:region:` can be retrieved from the bosh manifest
```
Comment (jameshochadel) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: Same comment as above re. moving the earlier in the sequence

**Review by jameshochadel — COMMENTED** [review]
This is a big improvement, thanks @markdboyd! I left two minor comments, otherwise this LGTM.

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 4]
```diff
@@ -62,15 +62,89 @@ ElasticSearch is corrupted or lost.
...
+AWS console and adjust the IAM role policy for the environment
+ingestor you are restoring in.
+
+You can find information about your ingestor VM by running:
+
+```shell
+bosh -d logsearch vms
+```
+
+For the VM with an instance name beginning with "kibana", take note of the value
```
Comment (markdboyd) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: Fix kibana -> ingestor

**Comment by markdboyd** [comment]
Thanks @jameshochadel! Addressed the feedback. Let me know what you think.

**Review by jameshochadel — APPROVED** [review]
Looks great, thanks Mark!

**Pull request merged**